src/styles/index.scss

```diff
@@ -1,3 +1,222 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+html,
+body,
+main,
+div,
+header,
+ul,
+li,
+a,
+h1,
+p,
+img {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  @include font($size: 10px, $weight: 400, $color: $main-accent-color);
+
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+button {
+  background-color: transparent;
+  color: inherit;
+  border-width: 0;
+  padding: 0;
+  cursor: pointer;
+}
+
+h1,
+p {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+ul,
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  @include flex(row, space-between, center);
+
+  max-width: 1200px;
+  margin: 0 auto;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo {
+    img {
+      @include set-size(40px, 40px);
+
+      display: block;
+    }
+  }
+
+  .nav {
+    &__list {
+      display: flex;
+    }
+    &__item:not(:last-child) {
+      margin-right: 20px;
+    }
+
+    &__link {
+      @include flex($direction: row, $items: center);
+
+      position: relative;
+      height: 60px;
+      color: $main-accent-color;
+      text-transform: uppercase;
+
+      &:hover {
+        color: $blue-accent-color;
+      }
+    }
+    .is-active {
+      color: $blue-accent-color;
+    }
+
+    .is-active::after {
+      @include set-size(100%, 4px);
+
+      content: '';
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      border-radius: 8px;
+      background-color: $blue-accent-color;
+    }
+  }
+}
+
+.main {
+  --number-of-columns: 1;
+
+  padding: 50px 40px;
+  &__container {
+    max-width: 944px;
+    display: grid;
+    grid-template-columns: repeat(var(--number-of-columns), 200px);
+    justify-content: center;
+    gap: 46px 48px;
+    position: relative;
+    margin: 0 auto;
+
+    @media (min-width: 488px) {
+      --number-of-columns: 2;
+    }
+
+    @media (min-width: 768px) {
+      --number-of-columns: 3;
+    }
+
+    @media (min-width: 1024px) {
+      --number-of-columns: 4;
+    }
+  }
+}
+
+.card {
+  @include flex($direction: column, $items: center);
+
+  max-width: 200px;
+  width: 100%;
+  border: 1px solid $light-white-color;
+  border-radius: $default-border-radius;
+  background-color: $white-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    @include set-size(160px, 134px);
+
+    display: block;
+  }
+
+  &__title {
+    @include font(12px, 500, 18px);
```

Comment:
Verify that the mixin 'font' is correctly defined in the imported mixins file. Incorrect or missing mixin definitions can cause compilation errors.